node_package/src/RSCPayloadContainer.ts

```
@@ -0,0 +1,100 @@
+    .replace(/<!--/g, '<\\!--')
+    .replace(/<\/(script)/gi, '</\\$1');
+}
+
+const RSCPayloadContainer = (
+  { chunkIndex, getChunkPromise }: RSCPayloadContainerInnerProps,
+): React.ReactNode => {
+  const chunkPromise = getChunkPromise(chunkIndex);
+  const chunk = React.use(chunkPromise);
+
+  const scriptElement = React.createElement('script', {
+    dangerouslySetInnerHTML: {
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)